### DISCUSSION TITLE
About datetimepicker

### DISCUSSION THREAD

**Discussion by ratneshupadhyay-kombee** [Power-Components/livewire-powergrid#1571] [BUG]
Filter::datetimepicker('created_at'), I Use this code for date range filter then i get this error 
Alpine Expression Error: pgFlatpickr is not defined
Uncaught ReferenceError: pgFlatpickr is not defined

**Comment by dansysanalyst** [comment]
https://livewire-powergrid.com/get-started/powergrid-configuration.html#flatpickr

**Reply by ratneshupadhyay-kombee** [reply]
i follow same things but its not working in my project i build a project from scratch
